competency-explorer/src/components/FilterBar/LevelFilters/LevelFilter/styles.scss

```
@@ -0,0 +1,18 @@
+.levelFilter {
+    padding: 0.1rem 0.25rem;
+    cursor: pointer;
+    margin-bottom: 0.5rem;
+    width: auto;
+    border-radius: 5px;
+
+    &:not(.hidden) {
+        background-color: #fff;
+        color: #333
+    }
+
+    &.hidden {
+        display: block !important;
```

Nothing bad happens when I remove the `!important` in prod; leftover from testing around?

```suggestion
        display: block;
```